tests/dags/providers/provider_api_scripts/test_inaturalist.py

```diff
@@ -35,14 +32,35 @@
 RECORD0 = JSON_RESPONSE[0]
 
 
+# Based on the small sample files in /tests/s3-data/inaturalist-open-data and on
+# minio in the test environment
+# Just checking raw record counts, assume that the contents load correctly as only
+# taxa has any logic outside literally loading the table using postgres existing
+# copy command.
+@pytest.mark.parametrize(
+    "file_name, expected",
+    [
+        ("00_create_schema.sql", [("inaturalist",)]),
+        ("01_photos.sql", [(37,)]),
+        ("02_observations.sql", [(32,)]),
+        ("03_taxa.sql", [(183,)]),
+        ("04_observers.sql", [(23,)]),
+    ],
+)
+def test_load_data(file_name, expected):
+    sql_string = (SQL_SCRIPT_DIR / file_name).read_text()
+    actual = PG.get_records(sql_string)
+    assert actual == expected
+
+
 def test_get_next_query_params_no_prior():
     expected = {"offset_num": 0}
     actual = INAT.get_next_query_params()
     assert expected == actual
 
 
 def test_get_next_query_params_prior_0():
-    expected = {"offset_num": INAT.batch_limit}
+    expected = {"offset_num": 10_000}
```

Comment:
Was there a reason this one was changed? `INAT.batch_limit` looks like the more appropriate choice.

Reply:
Yeah, I think I was ambivalently trying to have a test multi-task, but clarity and simplicity is better. I changed it back.